### ISSUE TITLE
Redirect to python-modernize/python-modernize is needed

### ISSUE THREAD

**Issue by techtonik** [mitsuhiko/python-modernize#33]
I've just done a double work by forking this repository istead of python-modernize/python-modernize.

Shouldn't there be a redirect to new location?

**Comment by daira** [comment]
Agreed, I'll try to arrange that. (Reparenting requires manual intervention of github admins, but is possible; I've seen it done for another project.)

**Comment by daira** [comment]
@mitsuhiko , do you have any objection to a reparenting? (mitsuhiko/python-modernize would become a fork of python-modernize/python-modernize rather than vice-versa.)

**Comment by techtonik** [comment]
Ping?

**Comment by daira** [comment]
Sorry, I lost track of this. Will get to it soon.

**Comment by hroncok** [comment]
Bump.

**Comment by jamesdbowman** [comment]
👍  +1

**Comment by jayvdb** [comment]
Bump.  This is higher in the Github search results for "python modernize"
